getting-started.md

```diff
@@ -37,7 +37,11 @@ bash Miniconda3-latest-Linux-x86_64.sh
 ```
 and then answer yes to all the questions!
 
-Log out and log back in again to activate the base conda environment.
+Log out and log back in again to activate the base conda environment. If this does not work, run:
+
+```
+source ~/.bashrc
```

Comment:
login shell runs ~/.bash_profile, not ~/.bashrc. We can copy the conda-init code to ~/.bash_profile with the following

cat ~/.bashrc >> ~/.bash_profile

Reply:
@bluegenes, @ctb added `echo ~/.bashrc >> ~/.bash_profile`